docs/known-issues.md

```diff
@@ -0,0 +1,35 @@
+
+# Known Issues and Limitations
+
+## Language systems and libraries
+* CPU affinity API is silently ignored
+* Language runtime base images are provided for a single version per language,
+  e.g. for Python 3.8 and not for Python 2 or Python 3.9
+## Kontain Monitor
+* There is no management plane to enumerate all running Kontain VMs
+* Snapshots are per VM. Coordinated snapshots for parent + children are currently not supported
+* Only a subset of `/proc/self` is implemented (the parts we see being used in Node.js, Python 3 and JVM 11)
+* `getrlimit` and `setrlimit` are not virtualized and are redirected to the host
+## Debugging
+* Stack trace through a signal handler is not useful
+* Attempt to access variables in thread local storage `p var` fails
+* Floating point registers are not supported

```

Comment:
Period.

This seems like a major shortcoming.  Doesn't (almost) every program use floating point someplace?  Is there a workaround we can give?